index.js

```diff
@@ -53,6 +53,7 @@ app.put('/user', (req, res) => {
 
 });
 
+//TODO: Request parameter validation, etc feeling should be an array
```

Comment:
i think in the requirements we have filtering by one feeling at a time, so string instead of array would be ok